### PR TITLE
gofmt code samples in README

### PR DESCRIPTION
**Pull request by y0ssar1an**
* `gofmt` all the code samples
* Fix syntax errors in code samples
* Fix typos in `README.md`
* Add syntax highlighting to all code samples

This supersedes the typo fix in #73.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,59 +137,55 @@ call passed handler function. Example:
...
 // Load the routes.
 func Load(router *vestigo.Router) {
-    router.Get("/debug/pprof/", Index)
-    router.Get("/debug/pprof/:pprof", Profile)
+	router.Get("/debug/pprof/", Index)
+	router.Get("/debug/pprof/:pprof", Profile)
 }
 
 // Index shows the profile index.
-func Index(w, http.ResponseWriter, r *http.Request) {
```
Comment (y0ssar1an) [inline README.md]: syntax error

File: README.md [hunk 2]
```diff
@@ -143,59 +137,55 @@ call passed handler function. Example:
...
-
 ```
 
 Note on wildcards: if you want to get the actual path matched by the wildcard
 you can perform `vestigo.Param("_name")` to get the matched path, example below:
 
-```
-    router.Get("/*", func(w http.ResponseWriter, r *http.Request) {
-        fmt.Println(vestigo.Param("_name"))
-    }
```
Comment (y0ssar1an) [inline README.md]: syntax error: missing closing paren

**Comment by y0ssar1an** [comment]
@husobee Your `.travis.yml` is broken and it's blocking PRs.

**Comment by husobee** [comment]
LGTM, thanks!

**Pull request merged**